frontend/src/pages/cert-manager/CertificatesPage/components/CertificatesTable.tsx

```diff
@@ -64,7 +64,7 @@ type Props = {
   handlePopUpOpen: (
     popUpName: keyof UsePopUpState<
       [
-        "certificate",
+        "certificateIssuance",
```

Comment:
Nit: Can we rename to `issueCertificate` to be consistent with `deleteCertificate`, `revokeCertificate`, etc.